### DISCUSSION TITLE
swapping the space (f and F) keys, provide unexpected behavior.

### DISCUSSION THREAD

**Discussion by leyloe** [helix-editor/helix#11029] [Troubleshooting]
### Summary

Swapping the space F keys results in file_picker_in_current_directory only being executed, instead of file_picker and file_picker_in_current_directory both being functional to their related keys.

```
[keys.normal.space]
f = "file_picker_in_current_directory" # This works as expected.
F = "file_picker" # This executes 'file_picker_in_current_directory' instead of 'file_picker'
```

### Reproduction Steps

 

I tried this:

```
[keys.normal.space]
f = "file_picker_in_current_directory"
F = "file_picker"
```

AND

```
[keys.normal.space]
f = "file_picker_in_current_directory"
S-f = "file_picker"
```

I expected this to happen:

execute 'file_picker'

Instead, this happened:

execute 'file_picker_in_current_directory'

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
nothing related to the file picker found
```

</details>

### Platform

linux

### Terminal Emulator

gnome terminal

### Installation Method

ubuntu ppa

### Helix Version

helix 24.3 (2cadec0b)

**Comment by kirawi** [comment]
I suspect you're not modifying the config at the right path. Are you opening the config via `:config-open`?

**Reply by leyloe** [reply]
I am using config open, why would i not

if i make other modifications, the configs function just fine. but this bugs out. i've even tried to purge the package, all configs, reinstalled and reinstated everything. no luck

**Reply by kirawi** [reply]
I tested and the config works as expected.

**Reply by leyloe** [reply]
> I tested and the config works as expected.

so is something with my package broken

**Reply by kirawi** [reply]
You can try with a completely blank config to see if there's a conflict. Else, I'd also make sure if it isn't working as you think it is. When pressing `space`, it shows the command associated with the key next to it.
